tools/src/main/java/org/apache/rocketmq/tools/command/consumer/SetConsumeModeSubCommand.java

```diff
@@ -64,7 +64,7 @@ public Options buildCommandlineOptions(Options options) {
         opt.setRequired(true);
         options.addOption(opt);
 
-        opt = new Option("n", "popShareQueueNum", true, "num fo queue which share in pop mode");
+        opt = new Option("q", "popShareQueueNum", true, "num of queue which share in pop mode");
```

Comment:
Why do we need change this? It seems would cause some compatibility issue.

Reply:
This option is already used by namesrv